src/components/fields/location-field/location-field.tsx

```diff
@@ -91,12 +108,11 @@ export const LocationField = (props: IGenericFieldProps<ILocationFieldSchema>) =
 				className={className}
 				locationInputPlaceholder={locationInputPlaceholder}
 				onChange={(e) => e.currentTarget.blur()}
-				onFocus={(e) => {
-					setShowLocationModal(true);
-					e.currentTarget.blur();
-				}}
+				onFocus={onFocus}
 				value={formValue?.address || ""}
 				errorMessage={error?.message}
+				disabled={disabled}
+				readOnly={readOnly}
 			/>
 			{!!formValue?.lat && !!formValue?.lng && (
 				<StyledStaticMap
```

Comment:
- StyledStaticMap should use `not-allowed` cursor if `disabled=true` 
- add `aria-disabled` to `src/components/shared/static-map/static-map.tsx` if component is disabled